other/restrict-secrets-store-csi-driver-loglevel-cve-2023-2878/restrict-secrets-store-csi-driver-loglevel-cve-2023-2878.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-secrets-store-csi-driver-loglevel
+  annotations:
```

Comment:
Missing annotations. See other examples.

Reply:
Annotations are [here](https://github.com/kyverno/policies/wiki/Kyverno-annotations). Look at some other samples for what we typically include. minversion isn't necessary unless it uses fields/features from versions within the past 3 releases.

Reply:
I do not feel like that answered my question, so I guess let me know if the added annotation is what you are looking for

Reply:
Ok, sorry. The two which are the most common are `kyverno.io/kyverno-version` (the version of Kyverno on which this policy was tested and confirmed to work) and `kyverno.io/kubernetes-version` (same but for Kubernetes itself). Others look ok.